### PR TITLE
Gcw 1544 specs

### PR DESCRIPTION
**Pull request by namrataukirde**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -90,4 +90,5 @@ group :test do
   gem "shoulda-callback-matchers"
   gem "codeclimate-test-reporter", require: nil if ENV["CI"]
   gem 'rspec_junit_formatter'
+  gem 'timecop'
```
Comment (trojanh) [inline Gemfile]: :+1: 
nice discovery.

File: spec/models/package_spec.rb [hunk 2]
```diff
@@ -491,6 +491,75 @@
...
+    let(:dispatched_location) { create :location, :dispatched }
+    let(:order) { create :order }
+    let(:orders_package) { create :orders_package, state: 'dispatched', package: package, order: order }
+    let(:dispatched_location) { create :location, :dispatched }
+
+    it 'creates dispatched packages location record against package if do not exist' do
+      expect{
+        package.create_dispatched_packages_location_from_gc(dispatched_location, orders_package.id, 1)
+      }.to change(PackagesLocation, :count).by(1)
+      expect(package.reload.packages_locations.first.location).to eq dispatched_location
```
Comment (steveyken) [inline spec/models/package_spec.rb]: perhaps remove repetition here:

    first_location = package.reload.package_locations.first.location
    expect(first_location).to eq(...)

File: spec/controllers/api/v1/packages_controller_spec.rb [hunk 3]
```diff
@@ -46,6 +51,37 @@ def test_orders_packages(package, stockit_request, count)
     end
   end
 
+ describe "PUT move_full_quantity" do
+    before { generate_and_set_token(user) }
+
+    before(:all) do
+      Timecop.freeze('2013-04-24'.to_date) { Time.zone.now.to_date }
```
Comment (steveyken) [inline spec/controllers/api/v1/packages_controller_spec.rb]: appear to be calling this with params and a block... does this make sense?

File: spec/models/package_spec.rb (hunk 2 above)
Comment (namrataukirde) [inline spec/models/package_spec.rb]: Sure @steveyken , Thanks

File: spec/controllers/api/v1/packages_controller_spec.rb (hunk 3 above)
Comment (namrataukirde) [inline spec/controllers/api/v1/packages_controller_spec.rb]: Whenver we will call Time.zone.now.to_date then it will return '2013-04-24'.to_date

**Pull request merged**